src/dotnet/APIView/APIViewWeb/Pages/Shared/_Layout.cshtml

```diff
@@ -68,6 +70,9 @@
                 crossorigin="anonymous"
                 integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=">
         </script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js"
```

Comment:
Or this?

Reply:
Shouldn't we just have one?

Reply:
This was supposed to be the jquery one. Updated.